app/models/user.rb

```diff
@@ -166,7 +166,8 @@ class User < ApplicationRecord
                        allow_nil: true,
                        if: :username_changed? # validate only when seller changes their username
 
-  validates :name, length: { maximum: MAX_LENGTH_NAME, too_long: "Your name is too long. Please try again with a shorter one." }
+  validates :name, length: { maximum: MAX_LENGTH_NAME, too_long: "Your name is too long. Please try again with a shorter one." },
+                   format: { without: /:/, message: "cannot contain ':' as it causes email delivery problems. Please remove any colons from your name and try again." }
```

Comment:
- Please avoid magic values and use a constant instead. (Ref. [other comment](https://github.com/antiwork/gumroad/pull/1751/files#r2449633050))
- This will break the app since it will cause existing records with names containing colons to be invalid too
  - This format validation should happen only:
    - on create
    - on update if it will change the name
  - Make absolutely sure to cover all these scenarios with tests